### ISSUE TITLE
Add info on showing the API (swagger ui)

### ISSUE THREAD

**Issue by karol-brejna-i** [karol-brejna-i/tigergraph-restpp-api#5]

**Comment by karol-brejna-i** [comment]
```bash
docker run -it -p 8080:8080 -e SWAGGER_JSON=/api.yaml -v /mnt/c/Users/kbrejna/git-projects/tigergraph-restpp-api/restpp.api.yaml:/api.yaml swaggerapi/swagger-uidocker run -it -p 8080:8080 -e SWAGGER_JSON=/api.yaml -v $PWD/restpp.api.yaml:/api.yaml swaggerapi/swagger-ui
```

**Comment by karol-brejna-i** [comment]
Closed with #10.
